docs/sphinx/user_guide/tutorial/matrix_transpose_local_array.rst

```diff
@@ -202,66 +202,92 @@ Here is the complete version including execution policy and kernel:
-          expression so the lambda expression argument lists are 
+.. note:: In this example, we need all five arguments in each lambda
+          expression so the lambda expression argument lists are
           the same. Another use case for the template parameter argument
           specification described here is to be able to pass only the
           arguments used in a lambda expression. In particular when we use
           multiple lambda expressions to represent a kernel, each lambda
           can have a different argument lists from the others.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-``RAJA::expt::launch`` Variants
+``RAJA::launch`` Variants
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``RAJA::expt::launch`` interface provides mechanisms to tile loops and use 
-*local arrays* in kernels to support algorithm patterns like the C-style kernel 
-above. When, using ``RAJA::expt::launch``, the ``RAJA_TEAM_SHARED`` macro is
-used to create a GPU shared memory array or a CPU stack memory array inside
-a kernel.
-
-``RAJA::expt::launch`` support methods for tiling over an iteration space
-using ``RAJA::expt::tile`` and ``RAJA::expt::loop_icount`` methods to tile
-loops and generate global iteration indices and local tile offsets.
-Moreover, lambda expressions for these methods will not be invoked for
-iterations outside the bounds of an iteration space when tile dimensions
-do not divide evenly the size of the iteration space; thus, no conditional
-checks on loop bounds are needed inside inner loops.
-
-A complete RAJA sequential CPU variant with kernel execution policy and 
+The ``RAJA::launch`` interface provides mechanisms to tile loops and use
+*local arrays* in kernels to support algorithm patterns like the C-style kernel
+above. When, using ``RAJA::launch``, the ``RAJA_TEAM_SHARED`` macro is
+used to create a GPU a static sized shared memory array when using the CUDA and HIP backends,
+static shared memory using SYCL is currently not supported. On the CPU, allocating
+``RAJA_TEAM_SHARED`` corresponds to allocating memory on the stack.
+Alternatively, one can allocated dynamic shared memory by specifying the amount of shared memory in
+the ``RAJA::LaunchParams`` struct. Dynamic shared memory is supported with all
+backends and will be demonstrated as our second example. On the CPU dynamic
+shared memory is mapped to heap memory and allocated via malloc at kernel launch.
+
+As a first example, we ilustrate the usage of static shared memory
+and the use of ``RAJA::launch`` tiling methods. RAJA tiling methods
+take a interation space in ``RAJA::tile`` and output tiles which
+the ``RAJA::loop_icount`` method can iterate over and generate
+global and local tile index offsets.  Moreover, lambda expressions for these
+methods will not be invoked for iterations outside the bounds of an iteration
+space when tile dimensions do not divide evenly the size of the iteration space;
```

Comment:
```suggestion
space when tile dimensions do not evenly divide the size of the iteration space;
```

Reply:
😮 split infinitive.... Jean-luc Picard would be proud

Reply:
Should we revert the change? I heard its may be okay depending on how formal the writing is.

Reply:
No need to revert. I was making a joke and was surprised @MrBurmark suggested it.

Reply:
I boldly split infinitives and beyond.